tests/od/main.c

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Tests od module.
+ *
+ * @author      Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "od.h"
+
+#define CALL(fn)    puts(#fn); fn
+
+static const char short_str[] = "AB";
```

Comment:
Oh well wait: the `long_str` contains bit patterns that have different results in unsigned and signed notation: `0xff`.